custom_components/openmediavault/binary_sensor.py

```diff
@@ -82,54 +82,26 @@ def __init__(self, omv_controller, inst, sensor):
 
         self._state = None
         self._icon = None
-        self._attrs = {ATTR_ATTRIBUTION: ATTRIBUTION}
-
-    @property
```

Comment:
Not sure that removing properties is a good idea, since those are overrides for extended class.

Reply:
Those _attr_values are also overrides the extended class, see here https://github.com/home-assistant/core/pull/53405 (and more PRs) for exmaple